xds/src/main/java/io/grpc/xds/XdsResourceType.java

```diff
@@ -85,12 +85,13 @@ public abstract class XdsResourceType<T extends ResourceUpdate> {
   // the resources that need an update.
   protected abstract boolean isFullStateOfTheWorld();
 
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/10847")
   public static class Args {
     final ServerInfo serverInfo;
     final String versionInfo;
     final String nonce;
     final Bootstrapper.BootstrapInfo bootstrapInfo;
-    final TlsContextManager tlsContextManager;
```

Comment:
Why hide TlsContextManager, it seems pretty generic?

Reply:
Stubby does not used it - they use LOAS.
TlsContextManager and its implementation are all marked as internal use only. All security things are internal. It doesn't seem we will expose it yet.